contents/handbook/engineering/feature-pricing.md

```diff
@@ -44,6 +44,12 @@ A good question to ask yourself here is, "If I were to switch away from PostHog,
 
 For example, if someone were to consider moving from PostHog to some other provider, cohorts would need to be manually recreated in the other provider, which would be tedious. However, something like Web Performance just happens and doesn't require any user involvement, so isn't sticky.
 
+### We should only charge users once per feature
```

Comment:
```suggestion
### Products can be used and paid for, without dependencies on other (paid) products
```

Rather than putting the emphasis on only charging users once, we should put the emphasis on being able to use a product fully without pricing dependencies on another product (like Raquel's changes below).
I think the current headline "only charge users once" can be misinterpreted as "we charge users only once for flags on the surveys product, in the flags product", but this is exactly a dependency we want to avoid by writing down this rule.